client/src/utils/constants/Tickets/CreateTicketField.ts

```diff
@@ -0,0 +1,31 @@
+import * as Yup from 'yup';
+
+export const createTicketField = [
+    { value: 'firstname', label: 'First name', img: null },
+    { value: 'lastname', label: 'Last name', img: null },
+    { value: 'subject', label: 'Subject', img: null },
+    { value: 'email', label: 'Email', img: null },
+    { value: 'destination', label: 'Destination', img: null },
+    { value: 'phone', label: 'Phone', img: null },
+    { value: 'fileName', label: 'File name', img: null },
+    { value: 'text', label: 'Text', img: null }
```

Comment:
all of the img passed as a null, think about remove it from here and paste in the rendering